resources/idlharness.js

```diff
@@ -1251,6 +1184,110 @@ IdlInterface.prototype.test_self = function()
 };
 
 //@}
+IdlInterface.prototype.test_immutable_prototype = function(type, obj)
+//@{
+{
+    if (typeof Object.setPrototypeOf !== "function") {
```

Comment:
This is related to [your question in gh-6246](https://github.com/w3c/web-platform-tests/pull/6246#issuecomment-308867046). As I mentioned in IRC, I'm not in a position to make a judgement call here, but I agree that we could use some clarity. I've opened gh-6266 to facilitate a discussion along these lines. Hopefully that results in a formally-documented policy.